src/unix/fsevents.c

```diff
@@ -21,9 +21,10 @@
 #include "uv.h"
 #include "internal.h"
 
-#if TARGET_OS_IPHONE
+#if TARGET_OS_IPHONE || ( MAC_OS_X_VERSION_MAX_ALLOWED < 1070 )
```

Comment:
Tiniest of style nits: no spaces after/before the parens. You don't need the parens at all, FWIW.